backend/frontend/templates/frontend/course_component_details/page.html

```diff
@@ -0,0 +1,92 @@
+                        Title: {{ course_component.title }}
+                    </p>
+                
+                    <div class="bold p-2 border-2 rounded-md ">
+                        Tag names: 
+                        {% with course_component.tag_names|length as array_length %}
+                            {% for item in course_component.tag_names %}
+                                {{ item }}
+                                {% if not forloop.last %},{% endif %}
+                            {% endfor %}
+                        {% endwith %}
+                    </div>
+                
+                    <div class="bold p-2 border-2 rounded-md ">
+                        <p class="bold">
+                            Status : {{board_status}}
+                        </p>
+                    </div>
+                </div>
+            
+                <div class="bold p-2 border-2 rounded-md grid gap-2">
+                
+                    <div class="bold p-2 border-2 rounded-md ">
+                        Assignees:
+                        <ul class="mx-8">
+                            <li class="bold">
+                            {{ course_component.recruit_user_names|first }}
+                            </li>
+                        </ul>
+                    </div>
+            
+                    <div class="bold p-2 border-2 rounded-md ">
+                        Reviewers: 
+                        <ul class="mx-8">
+                            {% for reviewer in course_component.reviewer_user_names%}
+                            <li class="bold">{{reviewer}}</li>
+                            {%endfor%}
+                        </ul>
+                    </div>
+                </div>
+                
+                <div class="grid bold p-2 border-2 rounded-md gap-2">
+                    <p class="bold p-2 border-2 rounded-md" >
+                        Start Date: {{course_component.start_time}}
+                    </p>
+
+                    <p class="bold p-2 border-2 rounded-md">
+                        Due Date: {{course_component.due_time}}
+                    </p>
+                </div>
+
+                <div class="bold p-2 border-2 rounded-md ">
+                    <p class="bold">
+                        Content URL: 
+                        <a class="{{styles.link}}" href="{{course_component.content_url}}">{{course_component.content_url}}</a>
+                    </p>
+                </div>
+
+                <div class="grid bold p-2 border-2 rounded-md gap-2">
+                    <h6>Link to your work</h6>
+                    
+                    <div>Current link submission: 
+                        {% if project.link_submission == None %}
+                            No link submitted yet
+                        {% else %}
+                        <a class="{{styles.link}}" href="{{course_component.link_submission}}">
```

Comment:
when linking to something external, can you get it to open in a new page? 

https://www.freecodecamp.org/news/how-to-use-html-to-open-link-in-new-tab/